index.css

```diff
@@ -55,10 +55,10 @@
     height: 5rem;
     background-color: var(--clr-whitesmoke);
```

Comment:
A suggestion as Vikram sir mentioned - not to mention the variable's value in the name.